apps/emqx/src/emqx_mountpoint.erl

```diff
@@ -17,6 +17,7 @@
 -module(emqx_mountpoint).
```

Comment:
shared topic should never be a mount-point.
so maybe this module should not be changed at all.

Reply:
when subscribing `$share/group/topic` on a listener with mount-point `mount/1`.
Should it be mounted to `$share/group/mount/1/topic`?